nondex-common/src/main/java/edu/illinois/nondex/common/ConfigurationDefaults.java

```diff
@@ -60,6 +60,9 @@ a copy of this software and associated documentation files (the
     public static final String DEFAULT_END_STR = "9223372036854775807";
     public static final long DEFAULT_END = new Long(ConfigurationDefaults.DEFAULT_END_STR);
 
+    public static final String PROPERTY_LOG = "logFile";
+    public static final String DEFAULT_LOG_STR = "log";
```

Comment:
log -> systematic.log
otherwise you may have duplicates and overwrite existing files - log is a fairly generic name that one could imagine is being used for many other things and there may be a file already called log created by users.